.changeset/quick-items-build.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/identity-apps-core": patch
```

Comment:
any reson to add this?

Reply:
Need for the release. Usual [practice](https://github.com/wso2/identity-apps/blob/master/docs/release/README.md).